CPAC/anat_preproc/anat_preproc.py

```diff
@@ -1168,7 +1168,6 @@ def freesurfer_preproc(wf, cfg, strat_pool, pipe_num, opt=None):
                                                       'wm_mask'],
                                         function=pick_tissue_from_labels_file),
                           name=f'anat_preproc_freesurfer_tissue_mask')
-    pick_tissue.inputs.include_ventricles = True
```

Comment:
I guess whether to include ventricles is implicit in which labels are chosen? :cool:

Reply:
Yes! Users can add ventricle labels to CSF label list directly.